<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Samples_Qt_TriggerCountClass</class>
 <widget class="QWidget" name="Samples_Qt_TriggerCountClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Samples_Qt_TriggerCount</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="lxcj">
       <property name="text">
        <string>连续采集</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rcf">
       <property name="text">
        <string>软触发</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="wcf">
       <property name="text">
        <string>外触发</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tz">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>显示比例:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="displayScale">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>20%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100%</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Set">
       <property name="text">
        <string>参数设置</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveImage">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>保存图片</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>拍摄数量:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pssl">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="MVScrollArea" name="scrollArea">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>882</width>
        <height>715</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="showImage">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MVScrollArea</class>
   <extends>QScrollArea</extends>
   <header location="global">mvscrollarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Samples_Qt_TriggerCount.qrc"/>
 </resources>
 <connections/>
</ui>
